internal/agent/hooks/kcrypt.go

```diff
@@ -16,6 +17,12 @@ func (k Kcrypt) Run(c config.Config) error {
 		return nil
 	}
 
+	// Config passed during install ends up here, so we need to read it
+	_ = machine.Mount("COS_OEM", "/oem")
+	defer func() {
+		_ = machine.Umount("/oem") //nolint:errcheck
+	}()
```

Comment:
So much for encrypting OEM I guess... :D Unlocking needs the config too right? (to find the challenger server url etc).